flowey/flowey_lib_hvlite/src/init_cross_build.rs

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//! Install dependencies and set environment variables for cross compiling
+
+use flowey::node::prelude::*;
+use std::collections::BTreeMap;
+
+flowey_request! {
+    pub enum Request{
+        InstallDeps {
```

Comment:
For clippy, this seems to be the case

Reply:
what is the error when you don't take the dep? just because taking this dep fixes the error, doesn't mean we're mapping the right semantics here per-se

Reply:
The error was something along the lines of gcc-... not installed. I think the env vars are not needed because it was being called in a build script somewhere that builds something in C that is never actually linked into anything.

Reply:
Ok